src/main/resources/tags/help/domain.tag

```diff
@@ -2,4 +2,6 @@ type: text
 aliases: domains, srv, srvrecord
 ---
 
-Bedrock doesn't work with SRV DNS records, so you'll have to use an A record instead. Java players will be able to connect fine with an A record.
\ No newline at end of file
+Bedrock doesn't work with SRV DNS records, so you'll have to use an A record instead. Java players will be able to connect fine with an A record.
+
+If your server is in a hosting company, and the address they assign you is not a string of numbers (raw IP), a CNAME record will also work for redirecting both Java and Bedrock players.
```

Comment:
Don't think the hosting part is necessary?
```suggestion
A CNAME record will also work for redirecting both Java and Bedrock players.
```
Up for debate, because I'm not sure.

Reply:
1+

CNAMEs work regardless of hosting provider or not. I don't really think any additional is necessary - if they're looking to setup a cname this is enough info, and if they need the solution a cname provides but they don't really understand it, this isn't the place to learn how they work

Reply:
Sounds good, committing suggestion into PR